client/src/Pos/Client/Txp/Util.hs

```diff
@@ -392,7 +392,11 @@ prepareTxRawWithPicker inputPicker utxo outputs (TxFee fee) = do
     when (totalMoney == mkCoin 0) $
         throwError $ GeneralTxError "Attempted to send 0 money"
 
-    let moneyToSpent = totalMoney `unsafeAddCoin` fee
+    moneyToSpent <- case integerToCoin (sumCoins [totalMoney, fee]) of
+        -- we don't care about exact number if user desires all money in the world
+        Left _  -> throwError $ NotEnoughMoney maxBound
```

Comment:
I'm unsure about this error. Are we sure that's descriptive enough for frontend to use? @DominikGuzei , does this make any difference for you folks? I don't think it does, but checking just in case 😉

Reply:
That's the same error they receive when user simply has no enough money on his account to spend. Here's indeed the exact same case

Reply:
Ah, perfect! Then simply remove the tmux change and I'm happy to merge! :)